README.md

```diff
@@ -0,0 +1,13 @@
+Follow setup to run app:
+
+Make sure you have nvm installed
+NPM version 6.14.8
+
+create .env in project
+Update in .env REACT_APP_WEATHER_API_KEY = "YOUR-KEY"
```

Comment:
this doesnt look good on github. we need bullet points and steps

Reply:
@Henshall done